src/main/resources/application.yaml

```diff
@@ -570,6 +570,15 @@ govnotify:
     requestNewHearingRequirements:
       legalRep:
         email: fd709bf4-095c-4734-8f88-9a9feba9fd81
+    recordRemissionDecision:
+      partiallyApproved:
+        legalRep:
+          email: 717b0290-36eb-4adf-a4c5-baacf59d379d
```

Comment:
This template is incorrect in GovNotify. You need to add placeholders for customer services telephone number and email address.

Reply:
Updated template in Gov notify